chart/epinio/templates/server.yaml

```diff
@@ -238,7 +238,7 @@ spec:
             - name: REGISTRY_CERTIFICATE_SECRET
               value: "{{ .Values.registry.certificateSecret }}"
             {{- end }}
-          image: splatform/epinio-server:{{ .Chart.AppVersion }}
+          image: {{ .Values.container.image }}:{{ .Values.container.tag }}
```

Comment:
This might as well go into a separate PR. Epinio needs to support the Rancher style for specifying images.
Example: https://github.com/rancher/charts/blob/dev-v2.6/charts/longhorn/longhorn/1.1.100/templates/deployment-driver.yaml#L23

https://github.com/epinio/epinio/issues/979

Reply:
After evaluation, I am wondering if that extra complexity is needed at this stage of the chart. Because it doesn't make sense to reuse the same structure for the values file defined on your link, and excepted the image published on dockerhub by docker like bash, we only  use the splatform/epinio-server  which could be overridden by just updatecli the value of .Values.container.image

Reply:
Dimitri covered this in hise pullrequest https://github.com/epinio/helm-charts/pull/38